src/main/java/InputHandler.java

```diff
@@ -0,0 +1,34 @@
+public class InputHandler {
+
+    private static final String TODO = "todo";
+    private static final String LIST = "list";
+    private static final String DEADLINE = "deadline";
+    private static final String EVENT = "event";
+    private static final String BYE = "bye";
+    private static final String MARK = "mark";
+    private static final String UNMARK = "unmark";
+
+    public static void handleInput() {
```

Comment:
Consider adding comments for all ur functions so it is more readable, as described in the code quality page on the module website